en/02_Developer_Guides/15_Customising_the_Admin_Interface/02_CMS_Architecture.md

```diff
@@ -338,31 +298,35 @@ can be used to pass state between route handlers and inspect the current
 history state. The `next` function invokes the next matching route. If `next`
 is called when there is no 'next' route, a page refresh will occur.
 
-### React router CMS sections
+#### React router CMS sections
 
 Similarly to page.js powered routing, the main point of registration for react routing
 sections is the `LeftAndMain::getClientConfig()` overridden method, which controls the main
-routing mechanism for this section. However, there are two major differences:
+routing mechanism for this section. However, there are three major differences:
+
+Firstly, where the page.js router uses the full path (e.g. 'admin/pages') as its route,
+react-router uses relative paths with nested routes. The main route is, by default, 'admin',
+which means the 'admin/pages' path would declare its route as 'pages'. This is added against the
+`reactRoutePath` key in the array returned from `LeftAndMain::getClientConfig()`.
 
-Firstly, `reactRouter` must be passed as a boolean flag to indicate that this section is
+Secondly, `reactRouter` must be passed as a boolean flag to indicate that this section is
 controlled by the react section, and thus should suppress registration of a page.js route
 for this section.
 
 ```php
 public function getClientConfig() 
 {
     return array_merge(parent::getClientConfig(), [
-        'reactRouter' => true
+        'reactRouter' => true,
     ]);
 }
 ```
 
-Secondly, you should ensure that your react CMS section triggers route registration on the client side
-with the reactRouteRegister component. This will need to be done on the `DOMContentLoaded` event
-to ensure routes are registered before window.load is invoked. 
+Lastly, you should ensure that your react CMS section triggers route registration on the client side
+with the `reactRouteRegister` component. This will need to be done on the `DOMContentLoaded` event
+to ensure routes are registered before window.load is invoked.
 
 ```js
-import { withRouter } from 'react-router';
```

Comment:
This HOC no longer exists in react-router. We do offer a replacement (outlined in the changelog) but it's a bit of a hack and should not be used for any _new_ components. It's only there to facilitate having a clear upgrade path and is clearly marked as a "push the can down the road" solution.